greeclimate/network.py

```diff
@@ -45,18 +38,19 @@ class IPInterface:
 class DeviceProtocolBase2(asyncio.DatagramProtocol):
     """Event driven device protocol class."""
 
+    _transport: Union[asyncio.transports.DatagramTransport, None] = None
+    _cipher: Union[CipherBase, None] = None
```

Comment:
I did a quick test using this branch. I can confirm that the broadcast message is encrypted with V1 Cipher. This means initially the cipher should be V1. During binding, if it times out, we should try V2 Cipher.

Reply:
That was a lucky guess on my side. Thanks for checking.